neuro_san/internals/graph/persistence/registry_manifest_restorer.py

```diff
@@ -38,7 +39,7 @@ class RegistryManifestRestorer(Restorer):
     for agent networks/registries.
     """
 
-    def __init__(self, manifest_files: Union[str, List[str]] = None):
+    def __init__(self, manifest_files: Union[str, List[str]] = None, agent_mapper: AgentNameMapper = None):
         """
```

Comment:
Add optional AgentNameMapper policy.
It will determine how we locate agent definition file given its name;
and how we find in-service agent name form its filepath.